web/src/components/text-input/text-field.tsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react'
+
+import { GenericField, InputProps } from './generic'
+
+export const TextField: React.FC<InputProps> = props => {
+  return <GenericField {...props} component="input" />
+}
+
+TextField.defaultProps = {}
```

Comment:
Let's remove all of the empty defaultProps

Reply:
done